src/melee/ft/chara/ftFox/ftfox.h

```diff
@@ -181,8 +181,13 @@ void ftFox_AppealS_OnTakeDamage(HSD_GObj* fighter_gobj);
 
 // Neutral Special - Blaster (SpecialN) //
 
+/// Get Fox/Falco's Item Hold Bone Position for Blaster GFX
```

Comment:
In general, briefs (the short description of a member) were moved to headers away from the implementation. Doxygen will take both at once, but it's generally considered better to put the usage information in the header and the implementation details in the C file.